src/index.html

```diff
@@ -11,6 +11,36 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
```

Comment:
Both forms are missing `action` and `method` attributes.
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/65851661/2448d2a7-8937-417e-9267-71480554d9cd)

Reply:
done! :)